File-Formats-toc.html

```diff
@@ -1670,6 +1670,7 @@
                     <li><a href="/file-formats/xlsio/faqs/how-to-upload-a-file-to-azure-blob-and-download-as-stream">How to upload a file to Azure blob and download as stream</a></li>
 					<li><a href="/file-formats/xlsio/faqs/how-to-set-rounded-corner-for-chart-in-excel-document">How to set rounded corner for chart in Excel document</a></li>
 					<li><a href="/file-formats/xlsio/faqs/how-to-find-and-replace-text-in-hyperlinks">How to find and replace text in hyperlinks</a></li>
+					<li><a href="/file-formats/xlsio/faqs/how-to-resolve-exception-when-using-Excel97to2003-with-large-number-of-row-column-index">How to resolve exception when using Excel97 to 2003 with large number of row column index</a></li>
```

Comment:
How to fix the ArgumentOutOfRangeException when accessing a large number of rows and columns?